ui/page/transaction/transaction_details_page.go

```diff
@@ -984,16 +984,16 @@ func (pg *TxDetailsPage) HandleUserInteractions() {
 // Part of the load.Page interface.
 func (pg *TxDetailsPage) OnNavigatedFrom() {}
 
-func initTxnWidgets(l *load.Load, transaction *sharedW.Transaction) transactionWdg {
+func initTxnWidgets(l *load.Load, wallet sharedW.Asset, transaction *sharedW.Transaction) transactionWdg {
 	var txn transactionWdg
-	wal := l.WL.SelectedWallet.Wallet
+	wal := wallet
```

Comment:
Just rename the `wallet sharedW.Asset` parameter to `wal sharedW.Asset`, or use `wallet` instead of `wal`.